packages/odyssey/src/scss/components/_modal.scss

```diff
@@ -15,7 +15,7 @@
 .ods-modal--overlay {
   display: flex;
   position: fixed;
-  z-index: 1;
+  z-index: 10;
```

Comment:
Good call. It shouldn't affect anything system-official, but any `z-index` change is probably breaking until we have the guidelines locked in.

Reply:
Does it make sense to hold off on this change until we finalize our guidelines on this? Otherwise isn't it likely that we'll introduce another breaking `z-index` change in a future release?